porcupine/plugins/python_venv.py

```diff
@@ -120,11 +122,16 @@ def _populate_menu(event: tkinter.Event[dirtree.DirectoryTree]) -> None:
     tree: dirtree.DirectoryTree = event.widget
     [item] = tree.selection()
     path = dirtree.get_path(item)
+
+    if path is None:
+        return
+
     project_root = dirtree.get_path(tree.find_project_id(item))
-    if not is_venv(path):
+    if project_root is None or not is_venv(path):
```

Comment:
- `project_root` can't be `None`, as a project item is never a dummy item.
- We can do the `is_venv()` check earlier, as in `if path is None or not is_venv(path)`